system_meta/urls.py

```diff
@@ -3,11 +3,34 @@
 from django.urls import include, re_path
 from rest_framework import routers
 
-from system_meta.views import IntegratedSystemViewSet, ProductViewSet
+from system_meta.views import (
+    IntegratedSystemViewSet,
+    ProductViewSet,
+    apisix_test_request,
+    authed_traefik_test_request,
+    traefik_test_request,
+)
 
 router = routers.DefaultRouter()
 
 router.register(r"integrated_system", IntegratedSystemViewSet)
 router.register(r"product", ProductViewSet)
 
-urlpatterns = [re_path("^", include(router.urls))]
+urlpatterns = [
+    re_path("^", include(router.urls)),
+    re_path(
+        r"^apisix_test_request/$",
+        apisix_test_request,
+        name="apisix_test_request",
+    ),
+    re_path(
+        r"^traefik_test_request/$",
+        traefik_test_request,
+        name="traefik_test_request",
+    ),
+    re_path(
+        r"^authed_traefik_test_request/$",
+        authed_traefik_test_request,
+        name="authed_traefik_test_request",
+    ),
```

Comment:
I moved these test request things under `/_/` which otherwise follows the structure of the `api` routes, but won't show up in the schema.

> While I'm on the subject, I notice that the APIs here don't appear to be prefixed with /api/{version}/. Don't have to address that in the is PR and I'm pretty sure I probably reviewed the PR that added that but we should keep to that standard.

This is done in the `unified_ecommerce/urls.py` which mounts these routes under `api/v0/meta`.